### PR TITLE
Permenant Protection Permission Issue

### PR DESCRIPTION
**Pull request by shthota77**
Permenant Protection Permission Issue

### CODE REVIEW

**Review by shthota77 — COMMENTED** [review]
typos

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 1]
```diff
@@ -102,3 +102,27 @@ background."
 Enable the required permission on Xiaomi devices.
 
 - Display pop-up windows while running in the background.
+
+
+## Unable to allow permission for 'Permanent protection' during onboarding, on some OEM devices
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
## Unable to allow permission for 'Permanent protection' during onboarding on some OEM devices
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 2]
```diff
@@ -102,3 +102,27 @@ background."
...
+
+**Applies to:** Specific OEM devices only
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
**Applies to:** Specific OEM devices only.
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 3]
```diff
@@ -102,3 +102,27 @@ background."
...
+
+-   **Xiaomi with Android 11** 
+
+Defender App asks for Battery Optimization / Permanent Protection permission on the devices as part of app onboarding, and by clicking on "Allow" it brings back an error that the permission couldn't be set. It only affects the last permission called "Permanent Protection".
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
Defender App asks for Battery Optimization/Permanent Protection permission on devices as part of app onboarding, and selecting **Allow** returns an error that the permission couldn't be set. It only affects the last permission called "Permanent Protection." 
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 4]
```diff
@@ -102,3 +102,27 @@ background."
...
+
+-   **Xiaomi with Android 11** 
+
+Defender App asks for Battery Optimization / Permanent Protection permission on the devices as part of app onboarding, and by clicking on "Allow" it brings back an error that the permission couldn't be set. It only affects the last permission called "Permanent Protection". 
+ 
+**Cause:**
+Xiomi changed the battery optimization permissions in Android 11. Defender is not allowed to configure this setting to ignore battery optimizations.
+
+**Solution:**
+We are working with OEMs for a solution to enable this permission form app onboarding screen. We will update the documentation as this is resolved.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
We are working with OEM to find a solution to enable this permission from the app onboarding screen. We will update the documentation when this is resolved.
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 5]
```diff
@@ -102,3 +102,27 @@ background."
...
+User can follow below steps to enable the same permissions form device settings.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
Users can follow these steps to enable the same permissions from the device settings: 
```

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 6]
```diff
@@ -102,3 +102,27 @@ background."
...
+We are working with OEMs for a solution to enable this permission form app onboarding screen. We will update the documentation as this is resolved.
+User can follow below steps to enable the same permissions form device settings.  
+
+1. Go to Settings on your device.
+2. Search for "Battery Optimization" and tap on "Battery Optimization".
+3. On the Special app access, click on "Battery Optimization".
+4. Change the Drop down to show all Apps.
+5. Locate “Microsoft Defender Endpoint” and select “Don’t Optimize”.
+
+Return to Defender onboarding screen, click on allow and you will be redirected to the dashboard screen.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]:
```suggestion
1. Go to **Settings** on your device.
2. Search for and select **Battery Optimization**.
3. In **Special app access**, select **Battery Optimization**.
4. Change the Dropdown to show **All Apps**.
5. Locate “Microsoft Defender Endpoint” and select **Don’t Optimize**.

Return to the Defender onboarding screen, select **Allow**, and you will be redirected to the dashboard screen.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shthota77, kindly commit the suggested changes if you agree with them. Many thanks.

File: microsoft-365/security/defender-endpoint/android-support-signin.md [hunk 7]
```diff
@@ -102,3 +102,27 @@ background."
...
+ 
+**Cause:**
+Xiomi changed the battery optimisation permissions in Android 11. Defneder is not allowed to configure this setting to ignore battery optimizations.
+
+**Solution:**
+We are working with OEMs for a solution to enable this permission form onboarding screen. We will update the documentation as this is resolved.
+User can follow the below steps to enable the same permissions form device settings.  
+
+1. Go to Settings on your device.
+2. Search for "Battery Optimisation" and tap on "Battery Optimisation".
```
Comment (Priyankag16) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]: Search battery optimisation

Comment (Priyankag16) [inline microsoft-365/security/defender-endpoint/android-support-signin.md]: Search battery optimisation

**Pull request merged**